l2tdevtools/review_helpers/linter.py

```diff
@@ -32,12 +33,13 @@ def CheckFiles(self, filenames, rcfile):
     """
     print('Running linter on changed files.')
     failed_filenames = []
+
     for filename in filenames:
       print('Checking: {0:s}'.format(filename))
 
-      command = 'pylint --rcfile="{0:s}" {1:s}'.format(rcfile, filename)
-      exit_code = subprocess.call(command, shell=True)
-      if exit_code != 0:
+      return_code = epylint.lint(filename, ['--rcfile={0:s}'.format(rcfile)])
```

Comment:
This makes pylint a source dependency (and its dependencies), why not use `pylint` and `pylint-3` instead?